.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - "6"
+  - "8"
+cache:
+  directories:
+    - /home/travis/build/technopagan/sqip/vendor
+before_install:
+  - npm install -g npm
```

Comment:
This is not needed as npm is provided with nvm ;-)

Reply:
Absolutely true. Thats a case of copy pasta 🙈